__tests__/app.test.js

```diff
@@ -111,3 +111,62 @@ describe("GET /api/articles/:article_id", () => {
 })
 
 
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201: adds comment to comments table with corresponding article_id", () => {
+    const newComment = {
+      username: 'butter_bridge',
+      body: 'Im really enjoying my time learning to code at Northcoders.'
+    }
+    return request(app)
+      .post("/api/articles/5/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.newComment.body).toBe('Im really enjoying my time learning to code at Northcoders.')
+        expect(body.newComment.comment_id).toBe(19);
+        expect(body.newComment.article_id).toBe(5);
+        expect(body.newComment.author).toBe('butter_bridge');
+        expect(body.newComment.votes).toBe(0);
+        expect(typeof body.newComment.created_at).toBe('string');
+      });
+    });
+    test('400: responds with an appropriate status and error message when provided without the correct properties', () => {
+      return request(app)
+        .post('/api/articles/5/comments')
+        .send({
+          colour: 'kat'
+        })
```

Comment:
You're testing that the required fields here are throwing an error. Sounds like two tests? Missing username, and then missing body?